applications/surgical_scene_recon/run_surgical_recon.py

```
@@ -0,0 +1,265 @@
+        "--coarse_iterations",
+        str(args.coarse_iterations),
+        "--num_frames",
+        str(args.num_frames),
+    ]
+
+    if args.no_masks:
+        cmd_train.append("--no_masks")
+
+    print(f"Running: {' '.join(cmd_train)}\n")
+
+    ret = subprocess.call(cmd_train)
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)